grommet-toolbox.config.js

```diff
@@ -26,6 +26,15 @@ export default {
         path.resolve(__dirname, 'src/scss'),
         path.resolve(__dirname, 'node_modules')
       ]
+    },
+    module: {
+      loaders: [
+        {
+          test: /\.jsx?$/,
+          loader: 'babel-loader',
+          include: /node_modules\/react-gravatar/
```

Comment:
it is a shame that gravatar is not providing that in their end. Should we file and issue against them and try to add that in their project? I can do that if you want.